canonical/canonicalwebteam.discourse
# Read href instead of string from masked links

## Done

- Fixed a bug to read links in `href` rather than the string of the `<a>` tag in metadata
- Reads masked links properly

## QA

- Go to https://ubuntu-com-14427.demos.haus/takeovers
- Search "Register for Ubuntu Summit 2024"
- Click on the primary CTA and check that you are redirected to https://ubuntu.com/summit?utm_campaign=takeover
- See that the `primary_url` in discourse is a masked link https://discourse.ubuntu.com/t/takeover-ubuntu-summit-20-years-register-view-content-hub/48012

Fixes [WD-15819](https://warthogs.atlassian.net/browse/WD-15819)

[WD-15819]: https://warthogs.atlassian.net/browse/WD-15819?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
